### ISSUE TITLE
CN62     how to increase DVMT on CN62 

### ISSUE THREAD

**Issue by czh6888** [MrChromebox/firmware#245]
How to increase DVMT on CN62 , I found that 96MB DVMT not enough smooth 4K monitor in macos. How to increase DVMT.

**Comment by MrChromebox** [comment]
how did you come to the conclusion that the DVMT is the issue?

it's not configurable, even in firmware. I had to hex-edit the MRC (ram init) blob to modify it

**Comment by czh6888** [comment]
framebuffer-stolenmem + framebuffer-fbmem + framebuffer-cursormem（Haswell Only）should less than DVMT.
If framebuffer-stolenmem + framebuffer-fbmem less than 64MB in HD5500 , Macos cannot recognize 4K 3840x2160.
It only can recognize 2560x1440.  
Now I set that framebuffer-stolenmem=64MB framebuffer-fbmem=31MB, Macos  can recognize 3840x2160. But I found that not enough smooth in Macos.
When I set framebuffer-stolenmem + framebuffer-fbmem > 96MB, Macos Crash. So I think that DVMT in coreboot is 96MB
But when I turn into Windows10, I don't know why Windows10 recognize DVMT 128MB. And Windows10 is smooth enough. So I think maybe increase DVMT can make Macos smooth.
